src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/pact/provider/ManageRecallsUIPactTest.kt

```diff
@@ -106,7 +104,7 @@ class ManagerRecallsUiAuthorizedPactTest : ManagerRecallsUiPactTestBase() {
 
   @State("a recall exists")
   fun `a recall exists`() {
-    recallRepository.save(recallWithPopulatedFields(::RecallId.zeroes(), nomsNumber, emptySet()))
+    recallRepository.save(fullyPopulatedRecall(::RecallId.zeroes()).copy(documents = emptySet()))
```

Comment:
And if we stick with documents needing to be empty then perhaps re-use this which I see called elsewhere? `fullyPopulatedRecallWithoutDocuments`?